docs/table/index.rst

```diff
@@ -99,15 +99,16 @@ You can get summary information about the table as follows::
      c    str1         Column
      d float64 m / s Quantity
 
-From within a Jupyter notebook, the table is displayed as a formatted HTML
-table (details of how it appears can be changed by altering the
-``astropy.table.default_notebook_table_class`` configuration item):
+From within a `Jupyter notebook <https://jupyter.org/>`_, the table is
+displayed as a formatted HTML table (details of how it appears can be changed
+by altering the ``astropy.table.conf.default_notebook_table_class``
```

Comment:
The way configuration works in `astropy` is that the `__init__.py` files of the subpackages define a `Conf` class and then create an instance of that class called `conf`. It would be possible to create a link to the attribute of the `Conf` class, but the users should be using the corresponding `conf` variable instead. I'm not sure if creating the link here would be helpful or confusing.

Reply:
I'd lean towards helpful. We link to class methods and attributes elsewhere and hopefully most users understand python's paradigm where unless it's explicitly stated, e.g. a ``classmethod``, things on classes should be accessed on an instance of the class, not the class itself.

Reply:
This is a bit different because the user is not expected to create new instances of the `Conf` class, they are only ever supposed to use the hardcoded `conf` variable. But if you still think it is a good idea to create the link then I can do it when I squash the commits.

Reply:
Thanks!